### PR TITLE
Project auth

### PR DESCRIPTION
**Pull request by petrabrinck**
Link to my deployed project: https://pb-authorization.netlify.com/

My reflections on how this week's project turned out:
Camilla and I struggled back and forth. We got my backend to work and her frontend to work and couldnt figure out why. Sitting close to Theresa and Susanna we got almost the same problems along the way but Camilla and I felt we needed an explanation as to how this really works.

Things I'd like to have clarified or explained in more detail: 
Damien showed us how to clean up the code and Camilla got help from Jennie so I think we have some kind of clue now :-)

### CODE REVIEW

File: frontend/src/components/Login.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+            })
+            .then(user => {
+                if (user['message']) {
+                    setErrorMsg(user.message)
+                }
+                else {
+                    window.localStorage.setItem('userId', user.userId)
+                    window.localStorage.setItem('accessToken', user.accessToken)
+                    //window.location.href = '/MemberPage'
+                    history.push('/MemberPage') /** Viktor */
```
Comment (JennieDalgren) [inline frontend/src/components/Login.js]: ⭐️

File: frontend/src/components/Login.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+                <h2>Welcome back - we've missed you! </h2>
+                <div className='loginContainer'>
+                    <label >Username/Email  </label>
+                    <input value={email} placeholder='Enter Email' type='email' name='email' onChange={event => { setEmail(event.target.value) }} required></input>
+
+                    <label>Password     </label>
+                    <input value={password} placeholder='Enter Password' type='password' name='password' onChange={event => { setPassword(event.target.value) }} required>
+                    </input>
+                    {errorMsg && <div className='error-message'> {errorMsg} </div>}
+                    {/* <label>Reset password here</label> */}
```
Comment (JennieDalgren) [inline frontend/src/components/Login.js]: remove unused code

File: frontend/src/components/MemberPage.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+            })
+            .catch(err => {
+                // console.log('ERROR', err)
+            })
+    }, [userId, accessToken])
+
+    const handleLogOut = () => {
+        window.location.href = '/' //return to start page
+        window.localStorage.clear() // clears data
+    }
```
Comment (JennieDalgren) [inline frontend/src/components/MemberPage.js]: ⭐️

File: frontend/src/components/Registration.js [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+// const URL = 'http://localhost:9000/register'
+// const URL = 'https://pb-auth-api.herokuapp.com/register'
+const URL = process.env.REACT_APP_API_URL || 'https://pb-auth-api.herokuapp.com/register'
+
+export const Registration = () => {
+    const [name, setName] = useState('')
+    const [email, setEmail] = useState('')
+    const [password, setPassword] = useState('')
+    // const [repeat, setRepeat] = useState('')
```
Comment (JennieDalgren) [inline frontend/src/components/Registration.js]: remove

File: frontend/src/components/Registration.js [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+                    <label >Email</label>
+                    <input value={email} placeholder='Enter Email' type='email' name='email' onChange={event => setEmail(event.target.value)} required></input>
+
+                    <label>Password</label>
+                    <input value={password} placeholder='Enter Password' type='password' name='password' onChange={event => setPassword(event.target.value)} required>
+                    </input>
+
+                    {/* <label>Repeat Password:</label>
+                    <input value={repeat} placeholder='Repeat Password' type='password' name='repeat' onChange={event => setRepeat(event.target.value)} required>
+                    </input> */}
```
Comment (JennieDalgren) [inline frontend/src/components/Registration.js]: remove

File: frontend/src/components/login.css [hunk 6]
```diff
@@ -0,0 +1,73 @@
+/*****LOGIN FORM********/
+
```
Comment (JennieDalgren) [inline frontend/src/components/login.css]: clean up css files and remove unused code

**Comment by petrabrinck** [comment]
Tack!
Ska städa 😁🙏🏽

**Pull request closed without merging**